codas/bld.bat

```diff
@@ -9,139 +9,5 @@ if errorlevel 1 exit 1
-copy build\adcp\edit\blkscan.exe %SCRIPTS%\ || exit 1
-copy build\adcp\edit\botmpas3.exe %SCRIPTS%\ || exit 1
-copy build\adcp\edit\fix_temp.exe %SCRIPTS%\ || exit 1
-copy build\adcp\edit\fix_hdg.exe %SCRIPTS%\ || exit 1
-copy build\adcp\edit\flag.exe %SCRIPTS%\ || exit 1
-copy build\adcp\edit\getmat.exe %SCRIPTS%\ || exit 1
-copy build\adcp\edit\last_85.exe %SCRIPTS%\ || exit 1
-copy build\adcp\edit\logscan.exe %SCRIPTS%\ || exit 1
-copy build\adcp\edit\set_top.exe %SCRIPTS%\ || exit 1
-copy build\adcp\edit\updscan.exe %SCRIPTS%\ || exit 1
-copy build\adcp\edit\dbupdate.exe %SCRIPTS%\ || exit 1
-copy build\adcp\edit\fillgap.exe %SCRIPTS%\ || exit 1
-copy build\adcp\edit\newflag.exe %SCRIPTS%\ || exit 1
-copy build\adcp\edit\setflags.exe %SCRIPTS%\ || exit 1
-copy build\adcp\edit\set_lgb.exe %SCRIPTS%\ || exit 1
-copy build\adcp\cal\arrdep.exe %SCRIPTS%\ || exit 1
-copy build\adcp\cal\refabsbt.exe %SCRIPTS%\ || exit 1
-copy build\adcp\cal\timslip.exe %SCRIPTS%\ || exit 1
-copy build\adcp\cal\rotate.exe %SCRIPTS%\ || exit 1
-copy build\adcp\cal\rotnav.exe %SCRIPTS%\ || exit 1
-copy build\adcp\cal\fix_bt_ss_bug.exe %SCRIPTS%\ || exit 1
-copy build\adcp\scanload\loadping.exe %SCRIPTS%\ || exit 1
-copy build\adcp\scanload\scanping.exe %SCRIPTS%\ || exit 1
-copy build\adcp\ping2mat\ping2mat.exe %SCRIPTS%\ || exit 1
-copy build\adcp\util\bt2tobt3.exe %SCRIPTS%\ || exit 1
-copy build\adcp\util\convadcp.exe %SCRIPTS%\ || exit 1
-copy build\adcp\util\convadc2.exe %SCRIPTS%\ || exit 1
-copy build\adcp\util\fixunits.exe %SCRIPTS%\ || exit 1
-copy build\adcp\util\getnav.exe %SCRIPTS%\ || exit 1
-copy build\adcp\util\lst_temp.exe %SCRIPTS%\ || exit 1
-copy build\adcp\util\lst_npings.exe %SCRIPTS%\ || exit 1
-copy build\adcp\util\lst_hdg.exe %SCRIPTS%\ || exit 1
-copy build\adcp\util\pgbin.exe %SCRIPTS%\ || exit 1
-copy build\adcp\util\set_bit.exe %SCRIPTS%\ || exit 1
-copy build\adcp\util\setpmask.exe %SCRIPTS%\ || exit 1
-copy build\adcp\util\lst_conf.exe %SCRIPTS%\ || exit 1
-copy build\adcp\util\lst_alfa.exe %SCRIPTS%\ || exit 1
-copy build\adcp\nav\edfix.exe %SCRIPTS%\ || exit 1
-copy build\adcp\nav\lst_btrk.exe %SCRIPTS%\ || exit 1
-copy build\adcp\nav\nav2dir.exe %SCRIPTS%\ || exit 1
-copy build\adcp\nav\putnav.exe %SCRIPTS%\ || exit 1
-copy build\adcp\nav\put_txy.exe %SCRIPTS%\ || exit 1
-copy build\adcp\nav\put_tuv.exe %SCRIPTS%\ || exit 1
-copy build\adcp\nav\ref2mat.exe %SCRIPTS%\ || exit 1
-copy build\adcp\nav\refabs.exe %SCRIPTS%\ || exit 1
-copy build\adcp\nav\sm2mat.exe %SCRIPTS%\ || exit 1
-copy build\adcp\nav\smoothr.exe %SCRIPTS%\ || exit 1
-copy build\adcp\nav\ubprint.exe %SCRIPTS%\ || exit 1
-copy build\adcp\nav\nmea_gps.exe %SCRIPTS%\ || exit 1
-copy build\ctd\load\load_bot.exe %SCRIPTS%\ || exit 1
-copy build\ctd\load\load_ctd.exe %SCRIPTS%\ || exit 1
-copy build\ctd\load\load_odf.exe %SCRIPTS%\ || exit 1
-copy build\ctd\util\conv_ctd.exe %SCRIPTS%\ || exit 1
+:: Move binaries to conda paths.
+move %PREFIX%\bin\* %SCRIPTS%\ || exit 1
```

Comment:
@efiring moving from `bin` to `%SCRIPTS%`, instead of copying from the build directory, reduces the final package size in half and ensure that these binaries are available on the Windows path.  I am not sure if this will break something that expectes these to be at the `bin` directory.

PS: Note that I removed the copy of the libs and headers to conda expected Paths .  It turns out that `pycurrents` can find them alright the way it is.